src/app/routes/utils/regex/utils/index.js

```diff
@@ -77,19 +77,6 @@ const homePageServices = [
   'zhongwen',
 ];
 
-export const getFrontPageRegex = services => {
-  let frontPages = services;
-  if (isLive()) {
-    frontPages = services.filter(
-      service => !homePageServices.includes(service),
-    );
-  } else {
-    frontPages = [];
-  }
-  const serviceRegex = getServiceRegex(frontPages);
-  return `/:service(${serviceRegex}):variant(${variantRegex})?:amp(${ampRegex})?:lite(${liteRegex})?`;
-};
-
 export const getHomePageRegex = services => {
```

Comment:
Is this out of scope for this work... but now that all homepages are live, and we're removing front pages altogether, should we also consider tidying up this regex. We no longer need to maintain a list of homepage services, nor differentiate between test & live. If we do remove this, can we also revert this test, as we only need to test it on a single environment: https://github.com/bbc/simorgh/pull/12035/files#diff-a4bd970f7c8f661885b8c1dbcfd95d5ad0bb867f52e5b7a2bf10a8b4888801c6